app/views/courses/index.html.erb

```diff
@@ -30,15 +30,21 @@
   <p class="govuk-body-s govuk-!-margin-bottom-6">You’ll be taken to a Google Form to fill in your course details.</p>
 <% end %>
 
-<%= render partial: 'course_table', locals: { courses: @self_accredited_courses } if @self_accredited_courses %>
+<% if @self_accredited_courses %>
+  <section data-qa="courses__courses-table-section">
+    <%= render partial: 'course_table', locals: { courses: @self_accredited_courses } %>
+  </section>
+<% end %>
 
 <% @courses_by_accrediting_provider.each do |accrediting_provider, courses| %>
-  <h2 class="govuk-heading-m">
-    <span class="govuk-caption-m">Accredited body</span>
-    <%= accrediting_provider %>
-  </h2>
-
-  <%= render partial: 'course_table', locals: { courses: courses } %>
+  <section data-qa="courses__courses-table-section">
+    <h2 class="govuk-heading-m">
```

Comment:
No need to add an extra level of HTML. I'd just add the data attribute on the `<h2>`

Reply:
NVM, I see the value of this now 👌